tests/bofire/data_models/specs/strategies.py

```diff
@@ -105,6 +110,57 @@
         "use_output_constraints": True,
     },
 )
+specs.add_valid(
+    strategies.ActiveLearningStrategy,
+    lambda: {
+        "domain": Domain(
+            inputs=Inputs(
+                features=[
+                    ContinuousInput(
+                        key="a",
+                        bounds=(0, 1),
+                    ),
+                    ContinuousInput(
+                        key="b",
+                        bounds=(0, 1),
+                    ),
+                ]
+            ),
+            outputs=Outputs(features=[ContinuousOutput(key="alpha")]),
+        ).model_dump(),
+        "acquisition_function": qNegIntPosVar(n_points=2048).model_dump(),
+        **strategy_commons,
+    },
+)
+
+specs.add_invalid(
```

Comment:
This does not hold anymore as invalid, but you should modify it to test if the error is thrown if the keys of the weights does not match the output feature keys.